app/views/smart_proxies/welcome.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for(:title, _("Smart Proxies")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("sitemap", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Smart Proxies') %></h1>
+  <p><%= _("The Smart Proxy is a project which provides a restful API to various sub-systems.
```

Comment:
1. "RESTful"
1. "subsystems" for consistency with line below
1. Add a couple of built-in examples (DNS, DHCP)